src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -342,7 +417,9 @@ impl<'a, 'tcx> Qualifier<'a, 'tcx, 'tcx> {
                         }
 
                         // Make sure there are no extra unassigned variables.
```

Comment:
I'm not sure this loop even works anymore?

Reply:
Nor am I...